index.html

```diff
@@ -15,6 +16,7 @@
                 <div class="todo-list-card">
                     <h2 class="todo-title">Todo List</h2>
                     <div id="items"></div>
+                    <p id="todoErrorText">Can't create empty todo!</p>
```

Comment:
"an empty todo" sounds better in my opinion

Reply:
Done